LetsDoStuff.WebApi/Controllers/CreateActivityRequest.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LetsDoStuff.WebApi.Controllers
+{
+    public class CreateActivityRequest
```

Comment:
It looks like a model that describes the creation of an activity. 

> HTTP request consists of 4 fundamental elements: A request line, zero or more header (General|Request|Entity) fields followed by CRLF, and a space preceding the CRLF (indicating the end of the header fields) and optionally a message body.

And

> A data model (or datamodel) is an abstract model that organizes elements of data and standardizes how they relate to one another and to the properties of real-world entities.

Reply:
This is not a regular model but view-model and I don't see a problem here. Let's keep it as it is

Reply:
Ok, I agree with you

Reply:
I was mistaken. That name completely describes a query model. I should learn MVC better. 
Sorry.